### PR TITLE
Add stopPath option

### LINKED ISSUE #2: Add `stopPaths` option
This is the only thing missing from this package being the exact inverse of `camelcase-keys`. I just didn't have time or interest to port this option over.

https://github.com/sindresorhus/camelcase-keys#stoppaths

### PR DESCRIPTION
**Pull request by Balastrong**
This PR fixes #2

### CODE REVIEW

**Comment by sindresorhus** [comment]
You need to update the TypeScript types too.

File: index.d.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export type DecamelizeKeys<
 	Separator extends string = '_',
 	Exclude extends readonly unknown[] = EmptyTuple,
 	Deep extends boolean = false,
+	StopPaths extends readonly string[] = EmptyTuple,
```
Comment (sindresorhus) [inline index.d.ts:36]: You are not actually using the `StopPaths` option in the types.

https://github.com/sindresorhus/camelcase-keys/blob/27e1e9dd25a865a911aaa9eb16cfa018438fc4b3/index.d.ts#L68

You also need to add a type test in https://github.com/sindresorhus/decamelize-keys/blob/main/index.test-d.ts

**Comment by Balastrong** [comment]
You're right, sorry, I think I'm a bit lost in that huge type. I might need some help or you can continue from there :)

**Comment by sindresorhus** [comment]
Closing for now. I don't have time to look into this.

**Pull request closed without merging**